### DISCUSSION TITLE
Removing editor styles when initializing the editor

### DISCUSSION THREAD

**Discussion by adavidson21** [GrapesJS/grapesjs#4692] [General]
Hey all,

This may be a basic question (I'm new to grapesjs and using it in an Angular 11 project), but I'm looking to remove the grey top and side bar from the editor when initializing the editor. 

I came across this CSS from the Getting Started section of the docs (and it works when I inspect and manually add it to the element with gjs-cv-canvas class)- 

```
/* Reset some default styling */
.gjs-cv-canvas {
  top: 0;
  width: 100%;
  height: 100%;
}
```

But, I can't seem to figure out how to accomplish this on initialization. Ideally, I'd be able to add it in here: 
```
const editor = grapesjs.init({
  container: '#gjs',
  fromElement: true,
  height: '300px',
  width: 'auto',
  storageManager: false,
  panels: { defaults: [] },
});
```

Any help would be awesome!

**Comment by ronaldohoch** [comment]
Hello and welcome!

If you `console.log(editor.Panels.getPanels())` you will see all Panels available.
![image](https://user-images.githubusercontent.com/2287371/198617220-bc1ce5b7-5ab8-44b4-ba36-436c2e61e4e9.png)

The id property is the name of Panel
![image](https://user-images.githubusercontent.com/2287371/198617598-776581e9-3b9e-4d11-ba44-5a7d8ac76249.png)

So, you can do:
```js
editor.on("load",()=>{
   editor.Panels.removePanel("commands");
})
```

**Reply by adavidson21** [reply]
Using the init in my original question, 

```typescript
const editor = grapesjs.init({
  container: '#gjs',
  fromElement: true,
  height: '300px',
  width: 'auto',
  storageManager: false,
  panels: { defaults: [] },
});
```

I'm able to get the canvas to look like this: 

![image](https://user-images.githubusercontent.com/25669951/198843462-940e5e47-3cd6-4576-b930-2c0233d50dc7.png)

But the issue I'm trying to fix is removing the grey top and side bar in that image so that I'm left with only the white canvas (with the goal being to add custom block/style managers).

By using `panels: { defaults: [] },`, it seems to be taking care of removing all of the panels from the editor, because when I use that log statement, this is the output: 

![image](https://user-images.githubusercontent.com/25669951/198843634-b0b1f50b-4dc1-40fa-907f-ef9fce1f32d8.png)

So I guess I'm more or less looking for something like the editor on load function, but that sets the CSS of the editor instead.

**Reply by artf** [reply]
I'm not sure why you can't simply use that CSS rule to reset the default styling

**Reply by adavidson21** [reply]
Yeah, I was able to use that rule like 

```scss
:host ::ng-deep .gjs-cv-canvas {
    top: 0;
    width: 100%;
    height: 100%;
}
```

Which worked perfectly, but was more or less looking to see if it was something I could pass in on init. Just didn't know if I was missing it in the docs!

**Reply by ronaldohoch** [reply]
this rule isn't deprecated in Angular 12+?
